web/src/components/layouts/layout.tsx

```diff
@@ -575,7 +575,9 @@ export default function Layout(props: PropsWithChildren) {
               </Button>
             </div>
           ) : null}
-          <main className="p-3">{props.children}</main>
+          <main className="h-[calc(100dvh-64px)] p-3 lg:h-dvh">
```

Comment:
We need to keep the magic to account for the top langfuse nav bar on mobile only -- I'd be okay with this as we'll be adding the shadcn sidebar soon which will result in a refactor of this file and will put us in a position to remove the magic once and for all :)